### ISSUE TITLE
[1.10.2] Can't craft obsidian ring

### ISSUE THREAD

**Issue by kaitoyuuki** [lumien231/Random-Things#133]
When attempting to craft the obsidian ring using an obsidian skull and a firecharge in an anvil, the GUI shows the obsidian ring in the output slot with a red zero over it, leading me to believe that it's trying to craft zero obsidian rings, leading to problems. Clicking on the ring causes the GUI to update, showing a red X over the "produces" arrow and no product in the output slot.

We have baubles installed on the server, I had plenty of levels to craft the item, and no error was produced in console on attempting to craft this item.

**Comment by lumien231** [comment]
I just tried it and it seems to work fine, do you have the same version of baubles installed on the server and the client?

**Comment by kaitoyuuki** [comment]
The issue has since been fixed, and we have no idea what fixed it, because neither baubles nor Random Things were updated.

**Comment by lumien231** [comment]
Weird^^.

**Comment by EnderofGames** [comment]
This problem has just started for us, Sky Factory 3 server. What's different is that I have successfully made two obsidian rings before the bug. 

A little more detail: made one about a week ago. Having proven its use, decided to make three more yesterday. I made three obsidian skulls and a lot of fire charges, then went to the anvil. It broke immediately after the first obsidian ring was made. I tried changing names, using other charge, the other skull, getting extra XP levels, restarting my client- though admittedly I have not yet asked for the server to be restarted. 

Picture included. In it you can see I have enough levels, also left F3 on by complete accident. Any attempt to make the '0 obsidian rings' will result in the arrow being covered in a red 'X', as minecraft typically does if there is no result in the 'anvilling'. 

Anything else I can do to find the source of this error? 

![2017-07-01_14 53 13](https://user-images.githubusercontent.com/17417666/27765427-a593be80-5e6d-11e7-9ccd-4c6baf8ed301.png)

**Comment by EnderofGames** [comment]
Didn't expect to get ahold of the server admin for a restart so quickly. About half an hour later, we restarted the server. Went to check and it worked, picture shows both I made (forgot to get an inbetween picture of the anvil working). Seems like a server restart cures this odd bug. 


![2017-07-01_15 50 19](https://user-images.githubusercontent.com/17417666/27765665-4208cd30-5e75-11e7-909e-9dc14e80fca3.png)
